praktikumsplaner-frontend/frontend/src/api/PraktikumsstellenService.ts

```diff
@@ -200,12 +200,60 @@ export default {
             );
         }
     },
+    updatePraktikumsstelle(
+        stelle: Praktikumsstelle,
+        loading: Ref<boolean> | undefined
+    ): Promise<void> {
+        if (this.isAusbildungsPraktikumsstelle(stelle)) {
+            if (loading !== undefined) {
+                loading.value = true;
+            }
+            return fetch(
+                `${API_BASE}${PRAKTIKUMSSTELLE_BASE}/ausbildung/${stelle.id}`,
+                FetchUtils.getPUTConfig(stelle)
+            )
+                .then((response) => {
+                    FetchUtils.defaultResponseHandler(response);
+                    useSnackbarStore().showMessage({
+                        message: "☑ Praktikumsstelle erfolgreich bearbeitet",
+                        level: Levels.SUCCESS,
+                    });
+                })
+                .finally(() => {
+                    if (loading !== undefined) {
+                        loading.value = false;
+                    }
+                });
+        } else if (this.isStudiumsPraktikumsstelle(stelle)) {
+            if (loading !== undefined) {
+                loading.value = true;
+            }
+            return fetch(
+                `${API_BASE}${PRAKTIKUMSSTELLE_BASE}/studium//${stelle.id}`,
+                FetchUtils.getPUTConfig(stelle)
+            )
+                .then((response) => {
+                    FetchUtils.defaultResponseHandler(response);
+                    useSnackbarStore().showMessage({
+                        message: "☑ Praktikumsstelle erfolgreich bearbeitet",
+                        level: Levels.SUCCESS,
+                    });
+                })
+                .finally(() => {
+                    if (loading !== undefined) {
+                        loading.value = false;
+                    }
+                });
+        } else {
+            throw new Error(
+                "Praktikumsstelle konnte nicht nach Typ kategorisiert werden."
+            );
+        }
```

Comment:
the diffrences is only the path in the url. there should be a way with less code